backend/.env

```diff
@@ -1,2 +1,3 @@
 source activate edp
```

Comment:
there are no actual changes in this file, so this should not be a part of the pull request.